src-arduino/examples/pn532_read/pn532_read.ino

```diff
@@ -0,0 +1,136 @@
+#include <SoftwareSerial.h>
+#include <PN532_SWHSU.h>
+#include <PN532.h>
+
+// This doesnt work for esp32s2 LOlE since there is no build for it.
+SoftwareSerial SWSerial(12, 11); // RX, TX
+PN532_SWHSU pn532swhsu(SWSerial);
+PN532 nfc(pn532swhsu);
+
+void setup(void)
+{
+
+  Serial.begin(115200);
+  while (!Serial)
+    ; // Do nothing if no serial port is opened (added for Arduinos based on ATMEGA32U4)
+  Serial.println("Hello!");
```

Comment:
hello!

Reply:
these files are test files for either the pn532 or the mfrc module so im kinda not bothering to change some of the debugging code, i will change some other stuff about it to make it easier to understand the examples tho